### PR TITLE
Update Cable to fix git info

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ branches:
     - /release.*/
     - appveyor
     - hunter
-clone_depth: 1
+clone_depth: 100
```
Comment (XhmikosR) [inline appveyor.yml]: Is this really needed?

**Comment by XhmikosR** [comment]
@chfast: I haven't followed cable completely, but is there any reason you are using `--abrev=40` and not a more sensible short one?

File: appveyor.yml (hunk 1 above)
Comment (chfast) [reply appveyor.yml]: To allow git access previous version tags `v0.X.Y...`.

Comment (XhmikosR) [reply appveyor.yml]: And what if a branch has like 200 commits?

This isn't foolproof. Only by cloning the whole repo you can be sure...

Comment (chfast) [reply appveyor.yml]: That's true, but it will still build. Just the project version will not have the distance from the latest release.

**Pull request merged**